provider/server.go

```diff
@@ -421,6 +421,11 @@ func (s *RawProviderServer) Configure(ctx context.Context, req *tfplugin5.Config
 	clientConfig, err := cc.ClientConfig()
 	if err != nil {
 		Dlog.Printf("[Configure] Failed to load config:\n%s\n", spew.Sdump(cc))
+		// TODO: improve the way of detecting this error
+		if err.Error() == "invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable" {
```

Comment:
The error is defined [here](https://github.com/kubernetes/client-go/blob/a099a3ff31eafc7018e64b8ed813e0f0650f4f38/tools/clientcmd/validation.go#L33). Could do: `errors.Is(err, clientcmd.ErrEmptyConfig)`

Reply:
Thanks! Didn't look too far when writing this :) will change it